docs/Changelog.md

```diff
@@ -1010,6 +1010,249 @@ opset_import {
 <dd>Constrain input and output types to float tensors.</dd>
 </dl>
 
+### <a name="DoWhile-1"></a>**DoWhile-1**</a>
+
+  Do-While Loop. This loop has multiple termination conditions:
+  
+  1) Trip count. Iteration count specified at runtime. Set by specifying
+     the input M. Optional. Set to empty string to omit.
+     Note that a static trip count (specified at graph construction time) can be
+     specified by passing in a constant node for input M.
+  2) Loop termination condition. This is a loop-carried depedency from the body
+     net that contains a single boolean result.
+  
+  The graph body will be run once, then run while the AND of the above conditions
+  is true. In the event that an optional condition is not specified, that
+  condition is treated as if it were always true.
+  
+    == Sample usage ==
```

Comment:
Would it be perhaps easier to parse the usage if we used one of the shorthand notations supported by the onnx?

Reply:
So the motivation of me using the direct graph proto here is that it's a representative example that's simultaneously human-interpretable and machine-interpretable, allowing for backends (etc.) to use this as a test. The other options we have are

1) The graph printer format, which there isn't a parser for yet (iiuc) so we lose the machine-readable aspect
2) Use the helper functions from helper.py. This hurts human readability since you essentially have to execute this meta-program (either in Python or in your head)

Thoughts? I'd love to make this more readable